### PR TITLE
Add January 2022 changelog

### LINKED ISSUE #1539: Changelog - January 28, 2022

### PR DESCRIPTION
**Pull request by mikenikles**
Close #1539.

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1540"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1540"></a>

### CODE REVIEW

**Review by jankeromnes — APPROVED** [review]
Great wording! 🎯 I like this highlight. 😁 ✨

One thing I'm never sure of is whether we should capitalize Projects and Prebuilds (as in, they're specific Gitpod concepts) or instead use projects and prebuilds (as in, they're commonplace concepts).

In the current state, I see that you've capitalized Prebuilds (which I read as a Gitpod concept 👍) but not projects (which I find just slightly confusing -- i.e. what do we mean here... repositories? big things you are working on? Gitpod's own feature? etc)

TL;DR -- In this text, I feel like I'd slightly prefer "Projects" capitalized, but this is definitely subjective so please feel free to disregard. 😁

**Review by jldec — APPROVED** [review]
Nicely worded - thanks @mikenikles !

**Comment by gtsiolis** [comment]
Thanks @mikenikles! I've rebased and added a new header image for this changelog entry in https://github.com/gitpod-io/website/pull/1540/commits/1180f2b35085cc76e0890069565419e316e4f2f5. Updated also the changelog entry date using today's date. Feedback is welcome! Cc @jankeromnes @jldec 

See also [relevant discussion](https://gitpod.slack.com/archives/C01LEBZ0EQ0/p1643650339334529?thread_ts=1643365170.442939&cid=C01LEBZ0EQ0) (internal).
